quarkchain/cluster/tests/test_grpc_client.py

```diff
@@ -42,13 +53,24 @@ def test_status_code(self):
         server0 = self.build_test_server(NormalServer, server_port1)
         server0.start()
 
-        resp0 = GrpcClient(client_host, server_port1).set_rootchain_confirmed_block()
+        block = RootBlock(
```

Comment:
how about add some minor block headers to the root block and check the content at the mock server side